main.go

```diff
@@ -958,6 +994,8 @@ func appendConfigFromFile(config *Config, path string) error {
 	}
 
 	if connString, ok := file.Get("database", "conn_string"); ok {
+		config.ConnString = connString
+		fmt.Println(config.ConnString)
```

Comment:
Is this `fmt.Println` intended?

Reply:
No thanks for catching that. It was just a test print I forgot to delete. Removed in the newest push.